plugins/action/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ add_library(dronecore_action SHARED
 
 include_directories(
     ${CMAKE_SOURCE_DIR}/core
-    ${CMAKE_SOURCE_DIR}/libs/include
+    SYSTEM ${CMAKE_SOURCE_DIR}/libs/include
```

Comment:
Just wondering why `SYSTEM` comes between `core` and `libs/include`. Does the order matter here? It doesn't seem so from what I understand in the [documentation](https://cmake.org/cmake/help/v3.0/command/include_directories.html), in which case I would put it before the directories.

It is a detail, but the way it is now seems to me like if it was supposed to be applied only for `libs/include`. So it is just for the sake of double-checking (I have never used `SYSTEM`).

Reply:
I only want to use it for `libs` which I consider "external" libraries. I had to add it because the mavlink headers had build warnings, and I don't want to go in there right now and fix it.